src/MoviesList/MoviesList.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import MovieDetailed from 'src/MovieDetailed';
+import {detailedMovieApi} from 'src/api/detailedMovieApi';
+import MovieCard from 'src/MovieCard';
+import cx from 'classnames';
+import globalStyles from '../../node_modules/bootstrap/dist/css/bootstrap.min.css';
+import styles from './MovieList.css';
+import { Link } from 'react-router-dom';
+
+class MoviesList extends Component {
+    constructor(props){
+        super(props)
+        this.state = {
+            singleMovieData:"",
+            isDetailedView: false
+        }
+    }
+
+    getYear(val) {
+        return new Date(val).getFullYear();
+    }
+
+    detailedMovie = (id) =>{
+        detailedMovieApi(id)
+        .then(result => {
+            this.setState({
+              isDetailedView: true,
+              singleMovieData: result
+            });
+        },
+        (error) => {
+            this.setState({
+                isDetailedView: true,
+                error
+            });
+        })
+    }
+
+    render() {
+        const {data : data_} = this.props;
+        const data = data_ || [];
```

Comment:
looks weird

Reply:
i mean both L40 and L41 lines, could you explain why?

Reply:
Removed after implementing routing.